src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1685,6 +1685,16 @@ public ResponseEntity<Void> setSensitivityAnalysisParametersValues(
         return ResponseEntity.ok().build();
     }
 
+    @PostMapping(value = "/studies/{studyUuid}/sensitivity-analysis/complexity")
+    @Operation(summary = "set sensitivity analysis parameters on study, reset to default ones if empty body")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis parameters are set")})
+    public ResponseEntity<Integer> fetchFiltersComplexity(
+            @PathVariable("studyUuid") UUID studyUuid,
+            @RequestBody(required = false) SensitivityAnalysisParametersInfos sensitivityAnalysisParametersValues,
```

Comment:
I think **SensitivityAnalysisParametersInfos** should be required otherwise you need to add an additional check before calling fetchFiltersComplexity.  the complexity calculation is basen on SensitivityAnalysisParametersInfos . So if it's not present no need to pursue.